bitrise.yml

```diff
@@ -1,28 +1,42 @@
-format_version: 5
+format_version: "11"
```

Comment:
why quotes here?

Reply:
I think this is the proper format, changing to int shows an error.